packages/ws-server/src/webSocketServer.ts

```diff
@@ -72,6 +74,19 @@ const relay: Relay = new RelayImpl(logger, register);
 const mirrorNodeClient = relay.mirrorClient();
 const limiter = new ConnectionLimiter(logger, register);
 const wsMetricRegistry = new WsMetricRegistry(register);
+
+const cpuUsageGauge = generateCpuGauge(register, {
```

Comment:
So by initialize an instance of cpuUsageGauge, the `collect()` function automatically run itself? If so, we should remove the instance refference `cpuUsageGauge` as it's not used anywhere else, and leave the `generateCpuGauge()` function on its own.